shell/components/Resource/Detail/Metadata/IdentifyingInformation/identifying-fields.ts

```diff
@@ -167,8 +167,8 @@ export const useResourceDetails = (resource: any): undefined | ComputedRef<Row[]
   };
 
   return computed(() => {
-    return details
-      .filter((detail: any) => !detail.separator)
+    return details.value
+      .filter((detail: any) => !detail.separator && detail.content !== undefined && detail.content !== null)
```

Comment:
I added more to this filter because I realized we were doing more filtering in the legacy masthead.

https://github.com/rancher/dashboard/blob/master/shell/components/DetailTop.vue#L75